modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties

```diff
@@ -120,9 +132,16 @@ request-string=Request String
 response-string=Response String
 scope-parameter-name=Scope Parameter Name
 search-...=Search...
+search-container=Search Container
+search-container-help=Access helper methods to retrieve Search Container values like the total results count.
 search-insights-help=When a search is performed, this widget displays the query and response strings that are sent to and returned from the search engine. Use this widget for testing search behavior.
 search-options-help=Configure additional search options in this page.
 search-web-configuration-name=Search Web
+search-results-display-context=Search Results Display Context
+search-results-display-context-help=Access helper methods to retrieve the Search Results widget configuration.
+search-results-portlet-instance-configuration-name=Search Results Configuration
+search-results-template-key-default=Search Results Template Key Default
+search-results-web-template-configuration-name=Search Results Web Template Configuration Name
```

Comment:
@rbohl When you get the chance, could you review the wording on these new keys? I'm mostly concerned about the `*-help` ones. These appear when creating a new Widget Template in the code editor panel. Thanks!

<img width="1677" alt="Screenshot 2020-01-10 09 43 07" src="https://user-images.githubusercontent.com/4496722/72174065-ba773000-338d-11ea-9c99-c3a2ee3637ac.png">

Reply:
:+1:  Made a few comments.